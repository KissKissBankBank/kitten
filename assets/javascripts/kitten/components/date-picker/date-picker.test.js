import React from 'react'
import renderer from 'react-test-renderer'
import { DatePicker } from './index'

describe('<DatePicker />', () => {
  let component

  beforeEach(() => {
    component = renderer.create(<DatePicker />).toJSON()
  })

  it('matches with snapshot', () => {
    expect(component).toMatchSnapshot()
  })
})
