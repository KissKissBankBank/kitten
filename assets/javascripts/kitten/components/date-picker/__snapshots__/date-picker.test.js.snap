// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DatePicker /> by default matches with snapshot 1`] = `
<div
  className="sc-htoDjs bqdhgJ"
>
  <div
    className="DayPickerInput"
  >
    <div
      className="sc-ifAKCX bCBibu"
    >
      <input
        autoComplete="off"
        className="sc-bwzfXH cmvcxy sc-EHOje kAxTfU"
        disabled={false}
        name="text"
        onBlur={[Function]}
        onChange={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        placeholder="dd/mm/yyyy"
        type="text"
        value=""
      />
      <span
        className="sc-bZQynM TJfWV"
        disabled={false}
      >
        ðŸ“…
      </span>
    </div>
  </div>
</div>
`;

exports[`<DatePicker /> with custom input matches with snapshot 1`] = `
<div
  className="sc-htoDjs bqdhgJ"
>
  <div
    className="DayPickerInput"
  >
    <input
      autoComplete="off"
      onBlur={[Function]}
      onChange={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      placeholder="dd/mm/yyyy"
      type="text"
      value=""
    />
  </div>
</div>
`;
